specification/batch/resource-manager/Microsoft.Batch/stable/2022-10-01/BatchManagement.json

```diff
@@ -6093,7 +6101,7 @@
     "CIFSMountConfiguration": {
       "type": "object",
       "properties": {
-        "username": {
+        "userName": {
```

Comment:
This is to fix a S360 issue that was detected. Our api was returning userName and not username. This was broken for a while